mlir/lib/Dialect/Linalg/IR/LinalgOps.cpp

```diff
@@ -149,6 +149,26 @@ static void fillStructuredOpRegion(OpBuilder &opBuilder, Region &region,
   // iterator_types is an auto-generated method.
 }
 
+/// Helper to create a typical indexing map for MatmulOp. Returns a list of
+/// AffineMap.
+static SmallVector<AffineMap>
+getDefaultIndexingMapsForMatmul(MLIRContext *context) {
```

Comment:
Why was this added here?

Reply:
I think the branch is broken, I don't see the original patch that got reverted. @shahidact can you crate a new branch and cherry-pick the two commits instead of rebasing? git often gets confused when rebasing.

Reply:
Closing this PR as another PR(https://github.com/llvm/llvm-project/pull/115319) is raised to this effect.